packages/communication-react/package.json

```diff
@@ -176,7 +176,8 @@
     "@eslint/js": "~9.27.0",
     "eslint-plugin-headers": "~1.3.1",
     "globals": "~16.2.0",
-    "@eslint/compat": "~1.2.9"
+    "@eslint/compat": "~1.2.9",
+    "jiti": "~2.4.2"
```

Comment:
Do we need to do any additional setup for jiti? (I don't see any usage only that is added to package.json files)

Reply:
No, it is just so that eslint can read TS directly